### ISSUE TITLE
Using the official wordnet release

### ISSUE THREAD

**Issue by danyaljj** [danyaljj/jwnl-prime#9]
Instead of using [this](https://github.com/danyaljj/jwnl-prime/blob/master/pom.xml#L54-L58) we can use this: 
```
      <dependency>
            <groupId>edu.princeton</groupId>
            <artifactId>wordnet-dict</artifactId>
            <version>3.1</version>
        </dependency>
```

The current version returns some wrong lemmas and suspect that it is an old wn. For example I tried `men` (which returns `men`): 
```
System.out.println(Dictionary.getInstance().getIndexWord(POS.NOUN, "men"));
```

Here is how I tried to change it: 
 - Added it to `pom.xml` 
 - Changed `<param name="dictionary_path" value="/wordnet/dict/"/>` to `<param name="dictionary_path" value="/wordnet-dict/"/>` in `map_properties.xml`. 

Although It didn't work. @christos-c  do you think the fact that the wordnet that you have used is "binary",  is important?

**Comment by christos-c** [comment]
@danyaljj, yes, `wordnet-binary` is a different version of the files that I generated using `net.didion.jwnl.utilities. DictionaryToMap`. If you want to use the official files you will have to convert them again.

**Comment by danyaljj** [comment]
Thanks I did it and it worked. Although I am observing a strange behavior.  WN gives me `men` as lemma of `men`.  Isn't it strange? I mean `men/men` are common words. Here is the usage:

```java 
        try {
            JWNL.initialize();
            System.out.println(Dictionary.getInstance().getIndexWord(POS.NOUN, "men"));
            System.out.println(Dictionary.getInstance().getIndexWord(POS.NOUN, "men").getLemma());
        } catch (JWNLException e) {
            e.printStackTrace();
        }
```

The output is 
```
[IndexWord: [Lemma: men] [POS: noun]]
men
```
(fyi @mssamoons )

**Comment by christos-c** [comment]
I don't know what to say given that the binary files were generated by the JWNL package itself. What happens if you use the standard files?

**Comment by mssammon** [comment]
just my 2 cents: I don't think you can replace the wordnet-binary with the wordnet-dict dependency: the former is a set of serialized data objects, and the latter is just plain files.
